docs/_get-started.asciidoc

```diff
@@ -0,0 +1,206 @@
+[[apm-setup-new]]
+== Get started (self-managed)
+
+TIP: The easiest way to get started with Elastic APM is by using our
+{ess-product}[hosted {es} Service] on {ecloud}.
+The {es} Service is available on AWS, GCP, and Azure.
+See <<apm-quickstart-new>> to get started in minutes.
+
+The APM Server binary receives performance data from your APM agents,
+validates and processes it, and then transforms the data into {es} documents.
+If you're on this page, then you've chosen to self-manage the Elastic Stack,
+and you now must decide how to run and configure the APM Server.
+There are two options, and the components required are different for each:
+
+* **<<setup-apm-server-binary,APM Server binary>>**
+* **<<setup-fleet-managed-apm,Fleet-managed APM Server>>**
+// * **<<setup-apm-server-ea,Standalone Elastic Agent-managed APM Server>>**
+
+[float]
+[[setup-apm-server-binary]]
+=== APM Server binary
+
+Install, configure, and run the APM Server binary wherever you need it.
+Depending on your use-case, APM Server can be run centrally or on edge machines.
+
+image::./images/bin-ov.png[APM Server binary overview]
+
+**Pros**:
+
+- Simplest self-managed option
+- No addition component knowledge required
+- YAML configuration simplifies automation
+
+**Supported outputs**:
+
+- {es}
+- {ess}
+- {ls}
+- Kafka
+- Redis
+- File
+- Console
+
+**Required components**:
+
+- APM agents
+- APM Server
+- {stack}
+
+**Configuration method**: YAML
+
+[float]
+[[setup-fleet-managed-apm]]
+=== Fleet-managed APM Server
+
+Fleet is a web-based UI in {kib} that is used to centrally manage {agent}s.
+In this deployment model, use {agent} to spin up APM Server instances that can be centrally-managed in a custom-curated user interface.
+
+NOTE: Fleet-managed APM Server does not have full feature parity with the APM Server binary method of running Elastic APM.
+
+image::./images/fm-ov.png[APM Server fleet overview]
+
+// (outputs, stable APIs)
+// not the best option for a simple test setup or if only interested in centrally running APM Server
+
+**Pros**:
+
+- Conveniently manage one, some, or many different integrations from one central {fleet} UI.
+
+**Supported outputs**:
+
+- {es}
+- {ess}
+
+**Required components**:
+
+- APM agents
+- APM Server
+- {agent}
+- {fleet}
```

Comment:
should rather be `Fleet Server` (Fleet is part of Kibana, so will always be there, but Fleet Server needs to be run as a component